packages/adminrouter/extra/src/Makefile.common

```diff
@@ -32,22 +32,23 @@ DEVKIT_DOCKER_OPTS := $(DEVKIT_BASE_DOCKER_OPTS) \
 
 .PHONY: clean-devkit-container
 clean-devkit-container:
-	-docker rm -vf adminrouter-devkit > /dev/null 2>&1
+	@# Let's suppress ALL the output:
+	@docker rm -vf adminrouter-devkit > /dev/null 2>&1 || true
```

Comment:
There is a shortcut for redirecting both `stdout` and `stderr` using `&>`, for example:

```
docker rm -vf adminrouter-devkit &> /dev/null || true
```